composer.json

```diff
@@ -5,32 +5,38 @@
   "license": "GPL-2.0-or-later",
   "require": {
     "php": ">=5.6",
-    "composer/installers": "^2.2"
+    "composer/installers": "^v1.12.0 || ^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^1.0.0",
+    "squizlabs/php_codesniffer": "^3.7",
+    "wp-coding-standards/wpcs": "dev-develop#fca9d9e"
   },
   "require-dev": {
-    "phpunit/phpunit": "^9.5",
-    "squizlabs/php_codesniffer": "^3.7",
-    "wp-coding-standards/wpcs": "dev-develop#fca9d9e",
     "wp-phpunit/wp-phpunit": "^6.1",
-    "yoast/phpunit-polyfills": "^1.0",
-    "phpcompatibility/php-compatibility": "^9.3",
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.7.2",
-    "phpstan/phpstan": "^1.10",
-    "szepeviktor/phpstan-wordpress": "^1.1",
-    "phpstan/extension-installer": "^1.2",
-    "slevomat/coding-standard": "^8.9"
+    "yoast/phpunit-polyfills": "^1.0"
   },
   "scripts": {
-    "lint": "phpcs --standard=phpcs.xml.dist",
-    "format": "phpcbf --standard=phpcs.xml.dist",
+    "lint": [
+      "composer --working-dir=build-cs install",
```

Comment:
Could we make this another script, maybe `pre-cs`

Reply:
That seems overkill do me, I don't see harm in this tiny bit of repetition. This way it's more connected with the next script and more readable, plus I don't see a reason why someone would want to run this command on its own.